api/namespace/applications/routes.py

```diff
@@ -59,70 +63,87 @@ def get(self):
         }
         return APPLICATIONS.search_applications(args), 200, response_headers
 
+    create_application_parser = reqparse.RequestParser()
+    create_application_parser.add_argument("account_id", location="json")
+    create_application_parser.add_argument("round_id", location="json")
+    create_application_parser.add_argument("fund_id", location="json")
 
-@applications_ns.route("", methods=["GET", "POST"])
-class ApplicationCreate(Resource):
-    """
-    Create a new application
-    """
-
-    @applications_ns.doc("create_application")
-    @applications_ns.expect(application_inbound)
+    @applications_ns.doc("post_application", parser=create_application_parser)
     @applications_ns.marshal_with(application_full, code=201)
     def post(self):
-        return APPLICATIONS.create_application(applications_ns.payload), 201
+        args = self.create_application_parser.parse_args()
+        account_id = args["account_id"]
+        round_id = args["round_id"]
+        fund_id = args["fund_id"]
+        application = APPLICATIONS.create_application(
+            account_id=account_id, fund_id=fund_id, round_id=round_id
+        )
+        return application, 201
 
 
-@applications_ns.route("/<application_id>", methods=["GET", "POST"])
-class Application(Resource):
-    """
-    Operations on a single application
-    """
+@applications_ns.route("/sections")
```

Comment:
why isnt this `/<application_id>/sections` ?

Reply:
Because (currently) afaik we can't dynamically update the callback url for individual PUT requests from the form runner, so we're getting the application_id from the metadata within the json in the request body